grails-app/conf/ImagesClientResources.groovy

```diff
@@ -21,8 +21,8 @@ modules = {
         resource url: [plugin: "images-client-plugin", dir: 'js/audiojs', file: 'audio.min.js']
     }
 
-    viewer {
-        dependsOn 'jquery', 'leaflet', 'leaflet_draw', 'font-awesome'
+    image_viewer {
+        dependsOn 'jquery', 'bootstrap', 'leaflet', 'leaflet_draw', 'font-awesome'
```

Comment:
I agree its not desirable, but yes currently. As it is the library uses HTML fragments provided by images-service for popup dialogs (subimaging, calibration). What I think we should change this so that it no longer uses the fragments and is just a straight JS library - but this was a stretch too far for this current refactoring. So I suggest we create an issue against images-client-plugin to do this refactor and then do it at the same same time as we tackle the changes of webservices in image-service. How does this sound ?